post-scan-actions/aws-python-slack-notification/terraform/README.md

```diff
@@ -0,0 +1,40 @@
+# Post Scan Action - Slack Notification
+
+After a scan occurs and a malicious file is detected, this example Lambda function sends out a notification to Slack using Amazon Simple Notification Service.
```

Comment:
Do we need the term "example"? "Sample" would be better, and removing it altogether would be better.
```suggestion
After a scan occurs and a malicious file is detected, this Lambda function sends out a notification to Slack using Amazon Simple Notification Service.
```